File: Logic/Multiples of 3 or 5.py
def solution(number):
    if number < 0:
        return 0
        
    contador = 0
    suma = 0
    
    while contador < number:
        if contador % 3 == 0 or contador % 5 == 0:
            suma += contador
        contador += 1
            
    return suma

File: Logic/Multiples of 3 or 5.js
function solution(number){
    if (number < 0) return 0
    let suma = 0;
    for (i=0; i<number; i++) {
        if (i % 3 == 0) {
            suma += i
        } else if (i % 5 == 0) {
            suma += i
        }
    }
    return suma
}